backend/src/main.ts

```diff
@@ -6,7 +6,7 @@ async function bootstrap() {
   const app = await NestFactory.create(AppModule);
   app.enableCors({
     origin: process.env.FRONTEND_URL,
-    methods: ['*'],
+    methods: ['*', 'DELETE'],
```

Comment:
I would remove the `'*'` and replace it with the methods we use (POST, DELETE, GET).
Because the `*` is not officially supported by express's [cors](https://github.com/expressjs/cors) package

> methods: Configures the Access-Control-Allow-Methods CORS header. Expects a comma-delimited string (ex: 'GET,PUT,POST') or an array (ex: ['GET', 'PUT', 'POST'])